src/android/io/branch/BranchSDK.java

```diff
@@ -221,16 +204,18 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
      */
     private void initSession(CallbackContext callbackContext)
     {
+
         this.activity = this.cordova.getActivity();
 
         Uri data = activity.getIntent().getData();
+
         if (data != null && data.isHierarchical()) {
             this.deepLinkUrl = data.toString();
         }
 
-        this.instance = Branch.getAutoInstance(this.activity.getApplicationContext());
-
+        this.instance = Branch.getInstance(this.activity.getApplicationContext());
```

Comment:
@aaustin updated PR, should be using `getAutoInstance` instead of `getInstance`.